components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/NetworkMessageEncoder.cs

```diff
@@ -135,11 +145,19 @@ public class NetworkMessageEncoder : IMessageEncoder {
                         encoder.WriteEncodeable(null, element);
                     }
                     encoder.Close();
+
+                    var body = Encoding.UTF8.GetBytes(writer.ToString());
+                    if (compressedPayload) {
+                        var stream = new MemoryStream(body);
```

Comment:
Please use using or try finally to ensure dispose

Reply:
Fixed